src/RealMeService.php

```diff
@@ -1053,35 +1080,24 @@ private function retrieveFederatedIdentity(OneLogin_Saml2_Auth $auth)
      * RealMeService.sync_with_local_member_database === true, then either create or update a local {@link Member}
      * object to include details provided by RealMe.
      */
-    private function syncWithLocalMemberDatabase() {
-        if($this->config()->sync_with_local_member_database === true) {
-            $member = DataObject::get_one("Member",
-                sprintf("RealmeSPNameID = '%s'", Convert::raw2sql($this->getAuthData()->SPNameID))
-            );
-
-            if(!$member){
-                $member = Member::create(["RealmeSPNameID" => $this->getAuthData()->SPNameID]);
-                $member->write();
-            }
+    private function syncWithLocalMemberDatabase()
+    {
```

Comment:
Logic here is gone - can we remove the method? Alternatively can we update the doc block to say why it's empty?